domains/ninjaclubreal.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "IDK",
+  "repo": "https://github.com/NINJACLUBREAL/mysite",
+  "owner": {
+    "username": "NINJACLUBREAL",
+    "email": "bee411769@gmail.com"
+  },
+"record": {
+        "A": [
+            "75.2.60.5"
+        ]
+  "URL": "https://ninjaclubdev.is-a.dev"
```

Comment:
Cannot use A records and URL records in conjunction with each other.

Reply:
ok will fix that this is my first time doing this

Reply:
> Cannot use A records and URL records in conjunction with each other.

i think I fixed it